### ISSUE TITLE
Ecommerce: Accessibility: Coupon code field has no label

### ISSUE THREAD

**Issue by briangrossman** [mitodl/mitxonline#516]
### Steps to Reproduce

- Begin a purchase and to the checkout page

### Expected Behavior

- A user would be able to effectively navigate the page with a screen reader

### Actual Behavior

- The `couponCode` &lt;input&gt; field isn't properly labeled. It's difficult to use with a screen reader. 

### Background from accessibility testers

This INPUT field has no label
` <input name="couponCode" type="text" id="coupon_code" class="form-control" autocomplete="given-name" aria-describedby="coupon_code_error" value="">`

Accessibility Error: no accessible name
WCAG 2.1 Success Criterion 1.1.1 Non-text Content (A)
Recommend: provide a label for the input element

Field should have a label per one of these methods: https://www.w3.org/WAI/tutorials/forms/labels/

**Screenshot of ARC Toolkit error**

![image](https://github.mit.edu/storage/user/923/files/e4a7a000-bbdd-11ec-8bd5-9e5c6c9d633f)
